notebooks/tr-TR/fine_tuning_code_llm_on_single_gpu.ipynb

```diff
@@ -0,0 +1,1127 @@
+{
```

Comment:
<p>not defterinde -&gt; "notebook'ta"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>